tools/ctl/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ target_compile_options(sof-ctl PRIVATE
 )
 
 target_include_directories(sof-ctl PRIVATE
+	"${SOF_ROOT_SOURCE_DIRECTORY}/../tools/include"
```

Comment:
I tested this and works in my local environment, but with @cgturner1 's test docker image (thesofproject/sof:20250410_no-alsa) with no system ALSA installed, the build fails as linkage still uses system libasound:

```

: && /usr/bin/cc -g  ctl/CMakeFiles/sof-ctl.dir/ctl.c.o -o ctl/sof-ctl  -lasound && :
/usr/bin/ld: cannot find -lasound: No such file or directory
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

Reply:
@kv2019i now fixed